app/src/main/res/layout/contract_detail_coverage_row.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
```

Comment:
Can this be LinealLayout?

Reply:
This layout is actually more or less identical to the previous one where I decided to go with `ConstraintLayout` over `LinearLayout`, hence my decision on that one still applies 🙂